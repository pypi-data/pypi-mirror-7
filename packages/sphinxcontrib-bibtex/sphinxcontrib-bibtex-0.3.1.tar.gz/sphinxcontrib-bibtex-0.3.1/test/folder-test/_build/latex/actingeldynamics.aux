\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\select@language{english}
\@writefile{toc}{\select@language{english}}
\@writefile{lof}{\select@language{english}}
\@writefile{lot}{\select@language{english}}
\newlabel{index-latex::doc}{{}{1}{}{section*.2}{}}
\@writefile{toc}{\contentsline {chapter}{\numberline {1}Part 1}{1}{chapter.1}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\newlabel{parts/part1::doc}{{1}{1}{Part 1}{chapter.1}{}}
\newlabel{parts/part1:contents}{{1}{1}{Part 1}{chapter.1}{}}
\newlabel{parts/part1:part-1}{{1}{1}{Part 1}{chapter.1}{}}
\@writefile{toc}{\contentsline {chapter}{\numberline {2}References}{3}{chapter.2}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\newlabel{index-latex:references}{{2}{3}{References}{chapter.2}{}}
\bibcite{1}{1}
\@writefile{toc}{\contentsline {chapter}{Bibliography}{5}{chapter*.4}}
\newlabel{bibitem:iwabuchi2010}{{2}{5}{\bibname }{section*.5}{}}
