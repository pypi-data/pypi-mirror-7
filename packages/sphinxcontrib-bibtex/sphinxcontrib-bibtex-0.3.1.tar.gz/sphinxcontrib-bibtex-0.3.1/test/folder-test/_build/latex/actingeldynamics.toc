\select@language {english}
\contentsline {chapter}{\numberline {1}Part 1}{1}{chapter.1}
\contentsline {chapter}{\numberline {2}References}{3}{chapter.2}
\contentsline {chapter}{Bibliography}{5}{chapter*.4}
