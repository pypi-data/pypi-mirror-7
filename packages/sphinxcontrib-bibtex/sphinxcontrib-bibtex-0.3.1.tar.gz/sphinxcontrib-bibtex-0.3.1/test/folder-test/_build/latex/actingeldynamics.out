\BOOKMARK [0][-]{chapter.1}{Part 1}{}% 1
\BOOKMARK [0][-]{chapter.2}{References}{}% 2
\BOOKMARK [0][-]{chapter*.4}{Bibliography}{}% 3
